packages/web-app-user-management/src/components/CompareSaveDialog.vue

```diff
@@ -1,20 +1,18 @@
 <template>
-  <div class="compare-save-dialog oc-p-s oc-width-1-1">
-    <div class="oc-flex oc-flex-between oc-flex-middle oc-width-1-1">
-      <span>{{ unsavedChangesText }}</span>
-      <div>
-        <oc-button :disabled="!unsavedChanges" @click="$emit('revert')">
-          <translate>Revert</translate>
-        </oc-button>
-        <oc-button
-          appearance="filled"
-          variation="primary"
-          :disabled="!unsavedChanges || confirmButtonDisabled"
-          @click="$emit('confirm')"
-        >
-          <translate>Save</translate>
-        </oc-button>
-      </div>
+  <div class="compare-save-dialog oc-p-s oc-width-1-1 oc-flex oc-flex-between oc-flex-middle">
+    <span>{{ unsavedChangesText }}</span>
```

Comment:
A bit more line height for the span would be nice so that the situations where buttons break into the next line are not too narrow.

Reply:
changed tag to p

Reply:
![image](https://user-images.githubusercontent.com/26169327/182135138-3e94219d-249b-4c10-a236-464f1863760b.png)

Reply:
That's too much. Bottom bar is jumpy when `p` and `div` are on the same line and you switch between the `saved` and `unsaved` states.